src/config/api.js

```diff
@@ -20,7 +20,7 @@ const errorMiddleware = {
   name: 'error-middleware',
   error: (payload) => {
     console.log('failed payload', payload);
-    if (payload.status === 401) {
+    if (payload.status === 401 || payload.status === 403) {
```

Comment:
`if (![401, 403].includes(payload.status))`

Reply:
I'm a fast learner 😄

Reply:
🤣 don't worry i need to work on this before we get the app outta beta so i'll be damn sure to fix this up